README.md

```diff
@@ -15,71 +15,75 @@ Here lies our Discord emojis!
+| ![](emoji/delivery.png)             | :delivery:              | Everfree Northwest Art ([Evergreen Dream](https://www.deviantart.com/okapifeathers)) |
+| ![](emoji/efnw.png)                 | :efnw:                  | Everfree Northwest Art ([Gemini Star](https://twitter.com/electrickeet)) |
+| ![](emoji/excited.png)              | :excited:               | Everfree Northwest Art ([JackOMT](https://www.deviantart.com/jackofmosttrades)) |
+| ![](emoji/facehoof.png)             | :facehoof:              | |
+| ![](emoji/float.png)                | :float:                 | Everfree Northwest Art ([Evergreen Dream](https://www.deviantart.com/okapifeathers)) |
+| ![](emoji/fly.png)                  | :fly:                   | Everfree Northwest Art ([Kelly the Drawing Unicorn](https://www.deviantart.com/kellythedrawinguni)) |
+| ![](emoji/frontpage.png)            | :frontpage:             | Everfree Northwest Art ([Evergreen Dream](https://www.deviantart.com/okapifeathers)) |
+| ![](emoji/gasp.png)                 | :gasp:                  | |
+| ![](emoji/happy.png)                | :happy:                 | |
+| ![](emoji/hello.png)                | :hello:                 | |
+| ![](emoji/hug.png)                  | :hug:                   | Everfree Northwest Art |
+| ![](emoji/huh.png)                  | :huh:                   | |
+| ![](emoji/joy-1.png)                | :joy-1:                 | Everfree Northwest Art |
+| ![](emoji/laugh.png)                | :laugh:                 | |
+| ![](emoji/lick.png)                 | :lick:                  | |
+| ![](emoji/lookup.png)               | :lookup:                | Everfree Northwest Art ([JaDeDJynX](https://www.deviantart.com/jadedjynx)) |
+| ![](emoji/marina.png)               | :marina:                | Everfree Northwest Art ([JaDeDJynX](https://www.deviantart.com/jadedjynx)) |
+| ![](emoji/marinaAngry.png)          | :marinaangry:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaEep.png)            | :marinasilly:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaEugh.png)           | :marinaeugh:            | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaRage.png)           | :marinarage:            | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaSad.png)            | :marinasad:             | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaShock.png)          | :marinashock:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaSilly.png)          | :marinasilly:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marinaSmile.png)          | :marinasmile:           | [Hollulu](https://twitter.com/meekcheep) |
+| ![](emoji/marina_high5_l.png)       | :marina_high5_l:        | FearDaKez |
+| ![](emoji/marina_high5_r.png)       | :marina_high5_r:        | FearDaKez |
```

Comment:
Would like to add highlighting to Marina to improve readability, especially in dark modes/tiny sizes